packages/server/src/db/index.js

```diff
@@ -82,20 +82,22 @@ async function getUser(id) {
             'users.email',
             'users.name',
             'users.role_id',
+            'users.tags',
+            'users.agency_id',
             'roles.name as role_name',
             'roles.rules as role_rules',
-            'users.agency_id',
             'agencies.name as agency_name',
             'agencies.abbreviation as agency_abbreviation',
             'agencies.parent as agency_parent_id_id',
             'agencies.main_agency_id as agency_main_agency_id',
             'agencies.warning_threshold as agency_warning_threshold',
             'agencies.danger_threshold as agency_danger_threshold',
-            'users.tags',
-            'users.tenant_id',
+            'tenants.id as tenant_id',
+            'tenants.display_name as tenant_name',
         )
         .leftJoin('roles', 'roles.id', 'users.role_id')
         .leftJoin('agencies', 'agencies.id', 'users.agency_id')
+        .leftJoin('tenants', 'tenants.main_agency_id', 'users.agency_id')
```

Comment:
This join is wrong. We should join based on `users.tenant_id = tenants.id` not the `main_agency_id`. Current user could be from a subagency that is not the `main_agency_id` for any tenant.

We will need to ensure that all users/agencies with null `tenant_id` are filled in before deploying this change. I believe that is already the case on staging, perhaps not on prod yet based on discussion in GOSTbusters meeting today?